recipes-bsp/bootfiles/rpi-config_git.bbappend

```diff
@@ -4,27 +4,27 @@ COMPATIBLE_MACHINE = "raspberrypi3"
 do_deploy_append() {
     if [ -z "${MENDER_ARTIFACT_NAME}" ]; then
         if [ -n "${KERNEL_IMAGETYPE}" ]; then
-            sed -i '/#kernel=/ c\kernel=${KERNEL_IMAGETYPE}' ${DEPLOYDIR}/bcm2835-bootfiles/config.txt
+            sed -i '/#kernel=/ c\kernel=${KERNEL_IMAGETYPE}' ${DEPLOYDIR}/${BOOTFILES_DIR_NAME}/config.txt
         fi
     fi
 
     if [ -n "${DISABLE_SPLASH}" ]; then
-        sed -i '/#disable_splash=/ c\disable_splash=${DISABLE_SPLASH}' ${DEPLOYDIR}/bcm2835-bootfiles/config.txt
+        sed -i '/#disable_splash=/ c\disable_splash=${DISABLE_SPLASH}' ${DEPLOYDIR}/${BOOTFILES_DIR_NAME}/config.txt
     fi
 
     # HDMI display rotation
     # 0 = none, 1 = 90cw, 2 = 180cw, 3 = 270cw, 0x10000 = hflip, 0x20000 = vflip
     if [ -n "${DISPLAY_ROTATE}" ]; then
-        sed -i '/#display_rotate=/ c\display_rotate=${DISPLAY_ROTATE}' ${DEPLOYDIR}/bcm2835-bootfiles/config.txt
+        sed -i '/#display_rotate=/ c\display_rotate=${DISPLAY_ROTATE}' ${DEPLOYDIR}/${BOOTFILES_DIR_NAME}/config.txt
     fi
 
     if [ -n "${ENABLE_SERIAL_CONSOLE}" ]; then
-        echo "" >> ${DEPLOYDIR}/bcm2835-bootfiles/config.txt
-        echo "dtoverlay=pi3-disable-bt" >> ${DEPLOYDIR}/bcm2835-bootfiles/config.txt
+        echo "" >> ${DEPLOYDIR}/${BOOTFILES_DIR_NAME}/config.txt
+        echo "dtoverlay=disable-bt" >> ${DEPLOYDIR}/${BOOTFILES_DIR_NAME}/config.txt
```

Comment:
Probably more of a question for @x85446 or @ygoyal18

Reply:
The bluetooth is disabled when serial is enabled. This is related to the same thing that Janne asked about: https://www.raspberrypi.org/documentation/configuration/uart.md Having the bluetooth and serial enabled simultaneously causes the serial to be filled with garbled messages. I think having `miniuart-bt` overlay could possibly allow both working at the same time, but it may hurt the bluetooth performance if I've understood correctly